CHANGELOG.unreleased.md

```diff
@@ -45,6 +46,7 @@
 - xo-server-backup-reports major
 - xo-server-netbox minor
 - xo-server-transport-email minor
+- xo-server minor
```

Comment:
I don't know if it's due to a rebase but `xo-server` is already in the list, so you can just remove it.